decidim-core/app/packs/stylesheets/decidim/modules/_cookie-consent.scss

```diff
@@ -0,0 +1,140 @@
+#cc-dialog-wrapper{
```

Comment:
I'm not sure if we can do it easily but I would try to reduce the amount of custom CSS we need here if possible.

I can achieve close to the same result with the following CSS without any custom CSS:
```html
<div id="cc-dialog-wrapper-alt" role="region">
  <div class="cc-dialog padding-vertical-1">
    <div class="row expanded">
      <div class="columns medium-12 large-8">
        <div class="cc-content">
          <div class="h5">
            Information about the cookies used on the website
          </div>
          <div>
            We use cookies on our website to improve the performance and content of the site. The cookies enable us to provide a more individual user experience and social media channels.
          </div>
        </div>
      </div>
      <div class="columns medium-12 large-4">
        <div class="text-center large-text-right">
          <button class="button margin-right-1 margin-bottom-0">Accept all</button>
          <button class="button hollow margin-right-1 margin-bottom-0">Essential only</button>
          <button class="button clear margin-right-1 margin-bottom-0" data-open="cc-modal">Settings</button>
        <div>
      </div>
    </div>
  </div>
</div>
```

Sure it's not quite the same (missing expanded mobile buttons, aligning the buttons center, etc.) but I'm just thinking based on this that if there's **any** way to reduce the amount of custom CSS, it would be great.